watcher/src/block_data_store.rs

```diff
@@ -152,6 +152,7 @@ impl BlockDataStore {
             block,
             block_contents,
             stored_block_data.signature,
+            None,
```

Comment:
We're gonna want the watcher to store metadata.

Reply:
I'm not sure we will. Block streaming will replace watcher and possibly mobilecoind (at least in fog servers), so I don't see the benefit of having watcher track metadata.

Reply:
I think the standalone watcher is still going to be desirable by people who want to monitor the blockchain for any discrepancies.

Reply:
OK, I'll add it

Reply:
Added to #2052